packages/flutter/lib/src/cupertino/search_field.dart

```diff
@@ -246,6 +250,11 @@ class CupertinoSearchTextField extends StatefulWidget {
   /// the comparison tool in https://github.com/flutter/platform_tests/.
   final EdgeInsetsGeometry prefixInsets;
 
+  /// Sets a prefix widget.
+  ///
+  /// Cannot be null. Defaults to [CupertinoIcons.search].
```

Comment:
nit: `[CupertinoIcons.search]` is not a widget.

Reply:
Understood 🙇

Reply:
@LongCatIsLooong Is this okay? a7ace8e